$OpenBSD$

Provide setLoopStatus() and setShuffle() dummies to fix
DESKTOP_APP_DISABLE_DBUS_INTEGRATION=ON build.
Pending PR: https://github.com/desktop-app/lib_base/pull/110

Index: Telegram/lib_base/base/platform/linux/base_system_media_controls_linux_dummy.cpp
--- Telegram/lib_base/base/platform/linux/base_system_media_controls_linux_dummy.cpp.orig
+++ Telegram/lib_base/base/platform/linux/base_system_media_controls_linux_dummy.cpp
@@ -46,6 +46,12 @@ void SystemMediaControls::setIsStopEnabled(bool value)
 void SystemMediaControls::setPlaybackStatus(PlaybackStatus status) {
 }
 
+void SystemMediaControls::setLoopStatus(LoopStatus status) {
+}
+
+void SystemMediaControls::setShuffle(bool value) {
+}
+
 void SystemMediaControls::setTitle(const QString &title) {
 }
 
